hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileDeleteHandler.java

```diff
@@ -68,7 +68,7 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         preValidate(transactionBody.fileID(), fileStore, context, true);
 
         var file = fileStore.getFileLeaf(transactionBody.fileID());
-        validateAndAddRequiredKeys(file.orElse(null), null, context);
+        validateAndAddRequiredKeysForDelete(file.orElse(null), null, context);
```

Comment:
Not part of this PR, but why does `ReadableFileStore.getFileLeaf()` return an `Optional`? We had a lengthy discussion at the start of the project and decided to return `null` in such cases. This code would be a lot simpler.

Reply:
ok I will fix it no worries. maybe I missed it same happens on consensus service as well.
I can also fix it here it is not an issue at all.

Reply:
created a separate issue for it https://github.com/hashgraph/hedera-services/issues/8198

Reply:
already merged